README.md

```diff
@@ -1,21 +1,112 @@
+2. Bump the version number in:
+    * `assets/macos/RSL.app/Contents/Info.plist` (change `CFBundleShortVersionString` to the new version number and increment `CFBundleVersion` by 1)
+    * `crate/ootr/Cargo.toml`
+    * `crate/rsl/Cargo.toml`
+    * `crate/rsl-cli/Cargo.toml`
+    * `crate/rsl-gui/Cargo.toml`
+    * `crate/rsl-updater/Cargo.toml`
+    * `crate/rsl-utils/Cargo.toml`
+    * `crate/rslbot/Cargo.toml`
+3. Push the changes
+4. On macOS (if releasing for macOS):
+    1. Install [Rust](https://rust-lang.org/) and [Python](https://python.org/).
+    2. Make sure your Rust compiler can build for both Intel and ARM:
+        ```sh
+        rustup target add x86_64-apple-darwin
+        rustup target add aarch64-apple-darwin
+        ```
+    3. Clone this repo.
+    4. In System Preferences → Sharing, make sure Remote Login is enabled and you have access.
+5. On Windows:
+    1. [Create a GitHub personal access token](https://github.com/settings/tokens/new) with the public_repo scope.
+    1. Install [Rust](https://rust-lang.org/) and [Python](https://python.org/).
+    2. Clone this repo.
+    3. In the repo clone, create a [JSON](https://json.org/) object at `assets\release-config.json` with the following entries:
+        * `"githubToken"`: The personal access token from step 1
+        * `"macHostname"`: A hostname or IP address (and if necessary the username, in `"username@hostname"` form) at which your Mac is reachable via SSH (optional, if not given no macOS app is included in the release)
+        * `"macRepoPath"`: The path to the clone of this repo on your Mac (optional, if not given no macOS app is included in the release)
+        * `"releaseNotesEditor"`: A command that will be called with `--wait` and the path to an empty file for writing the release notes (optional, defaults to `"C:\\Program Files\\Microsoft VS Code\\bin\\code.cmd"`)
+        * `"repoOwner"`: The GitHub user or org where the release should be published (optional, defaults to `"matthewkirby"`)
+        * `"repoName"`: The name of the repository where the release should be published (optional, defaults to `"plando-random-settings"`)
+    4. In the repo, run `cargo run --release --package=rsl-utils --bin=rsl-release`. You can add the following command-line options, but must separate them with a `--` (so they're interpreted as `rsl-release` options, not `cargo` options):
+        * To see output of build commands as they are happening, use `--verbose`. Note that this may considerably slow down the release script since it won't run multiple build commands in parallel.
+        * To create the GitHub release as a draft but not publish it, use `--no-publish`.
```

Comment:
I couldn't get the --no-publish flag to work. Can you show an example? Is it this?

cargo run --release --package=rsl-utils --bin=rsl-release -- --verbose -- --no-publish

Reply:
The `--` between `--verbose` and `--no-publish` is the issue here. I should clarify that.